modules/default.nix

```diff
@@ -100,29 +100,38 @@ inputs: {
   extraLuaPackages = ps: map (x: ps.${x}) vimOptions.luaPackages;
   extraPython3Packages = ps: map (x: ps.${x}) vimOptions.python3Packages;
 
+  luaConfig =
+    if vimOptions.byteCompileLua
+    then pkgs.runCommandLocal "init.lua" {text = vimOptions.builtLuaConfigRC;} "${pkgs.luajit}/bin/luajit -bd -- - $out <<< \"$text\""
```

Comment:
`pkgs.luajit` should be the unwrapped neovim's `.lua` attribute

Reply:
What does that mean

Reply:
Oh

Reply:
The lua package doesn't provide luajit though

Reply:
```nix
$ nix build .#neovim-unwrapped.lua
$ ls result/bin
lua  luajit  luajit-2.1.1713773202
```

Reply:
```bash
$ nix build github:nix-community/neovim-nightly-overlay#neovim.lua
$ ls result/bin
lua  luajit  luajit-2.1.1713773202
```